### ISSUE TITLE
Test failures related to non-English locale

### ISSUE THREAD

**Issue by eserte** [houseabsolute/perl-code-tidyall#109]
On some of my smokers t/Git.t fails, probably because a non-English locale is in effect (here: `LC_ALL=de_DE.ISO8859-1`):
```
...
[applying the following plugins: +TestHelper::Plugin::UpperText]
[checked] bar.txt (+TestHelper::Plugin::UpperText)

    #   Failed test 'branch is ahead of origin'
    #   at /usr/home/cpansand/.cpan/build/2021112618/Code-TidyAll-0.79-2/t/lib/TestFor/Code/TidyAll/Git.pm line 399.
    #   (in TestFor::Code::TidyAll::Git->test_git)
    #                   'Auf Branch master
    # Ihr Branch ist vor 'origin/master' um 1 Commit.
    #   (benutzen Sie "git push", um lokale Commits zu publizieren)
    # nichts zu committen, Arbeitsverzeichnis unver\xE4ndert
    # '
    #     doesn't match '(?^:Your branch is ahead)'
    # Looks like you failed 1 test of 2.

#   Failed test 'untidy file and attempt to commit it via commit -a'
#   at /usr/home/cpansand/.cpan/build/2021112618/Code-TidyAll-0.79-2/t/lib/TestFor/Code/TidyAll/Git.pm line 96.
#   (in TestFor::Code::TidyAll::Git->test_git)

    #   Failed test 'branch is ahead of origin'
    #   at /usr/home/cpansand/.cpan/build/2021112618/Code-TidyAll-0.79-2/t/lib/TestFor/Code/TidyAll/Git.pm line 399.
    #   (in TestFor::Code::TidyAll::Git->test_git)
    #                   'Auf Branch master
    # Ihr Branch ist vor 'origin/master' um 1 Commit.
    #   (benutzen Sie "git push", um lokale Commits zu publizieren)
    # nichts zu committen, Arbeitsverzeichnis unver\xE4ndert
    # '
    #     doesn't match '(?^:Your branch is ahead)'
    # Looks like you failed 1 test of 4.

... etc ...
    #   Failed test 'commit shows bar.txt as still modified'
    #   at /usr/home/cpansand/.cpan/build/2021112618/Code-TidyAll-0.79-2/t/lib/TestFor/Code/TidyAll/Git.pm line 211.
    #   (in TestFor::Code::TidyAll::Git->test_precommit_stash_issues)
    #                   'Auf Branch master
    # \xC4nderungen, die nicht zum Commit vorgemerkt sind:
    # 	ge\xE4ndert:       bar.txt
    # 
    # Unversionierte Dateien:
    # 	baz.txt
    # 
    # keine \xC4nderungen zum Commit vorgemerkt
    # '
    #     doesn't match '(?^:modified:\s+bar\.txt)'
    # Looks like you failed 1 test of 4.

...
```

**Comment by autarch** [comment]
Thanks for the bug report. My enthusiasm for fixing bugs in this distro is a bit low, since I think it's a bit of a dead end (see https://blog.urth.org/2020/04/25/the-real-dirt-on-tidyall/). But I'm happy to review a PR that fixes this.

Sorry in advance that you're about to get this same message three times, but I want this visible in each of the bugs you reported so others can see this.

**Comment by eserte** [comment]
It looks like this problem is fixed in 0.82, so closing this issue.
